app/experimenter/normandy/tasks.py

```diff
@@ -172,8 +172,8 @@ def update_firefox_versions(experiment, recipe_data, filter_objects):
     changed_data = {}
 
     if versions := filter_objects.get("version"):
-        min_version = min(versions["versions"]) * 1.0
-        max_version = max(versions["versions"]) * 1.0
+        min_version = str(min(versions["versions"]) * 1.0)
+        max_version = str(max(versions["versions"]) * 1.0)
```

Comment:
Oooohh multiplying by `1.0` to convert it to a float.

More idiomatic way is to just use `float` like

`str(float(min(versions["versions"])))`

Alternatively you could just use a format string like

`f"{min(versions['versions'])}.0"`

but then you'd either have to be sure it's an `int` or wrap it in an `int()` call maybe like

`f"{int(min(versions['versions']))}.0"`

Iunno there's a bunch of ways to do it, but I'd just use casts instead of multiplication to force it to the type you want 👍